docs/src/api.md

```diff
@@ -1,26 +1,80 @@
+  - Original NUTS with slice sampling from [hoffman2014no](@Citet): `HMCKernel(Trajectory{SliceTS}(integrator, ClassicNoUTurn()))`
+  - Generalised NUTS with slice sampling from [betancourt2017conceptual](@Citet): `HMCKernel(Trajectory{SliceTS}(integrator, GeneralisedNoUTurn()))`
+  - Original NUTS with multinomial sampling from [betancourt2017conceptual](@Citet): `HMCKernel(Trajectory{MultinomialTS}(integrator, ClassicNoUTurn()))`
+  - Generalised NUTS with multinomial sampling from [betancourt2017conceptual](@Citet): `HMCKernel(Trajectory{MultinomialTS}(integrator, GeneralisedNoUTurn()))`
+
+### Adaptor (`adaptor`)
 
-## Functions
+  - Adapt the mass matrix `metric` of the Hamiltonian dynamics: `mma = MassMatrixAdaptor(metric)`
+    
+      + This is lowered to `UnitMassMatrix`, `WelfordVar` or `WelfordCov` based on the type of the mass matrix `metric`
 
-```@docs
-sample
+  - Adapt the step size of the leapfrog integrator `integrator`: `ssa = StepSizeAdaptor(δ, integrator)`
+    
+      + It uses Nesterov's dual averaging with `δ` as the target acceptance rate.
+  - Combine the two above *naively*: `NaiveHMCAdaptor(mma, ssa)`
+  - Combine the first two using Stan's windowed adaptation: `StanHMCAdaptor(mma, ssa)`
+
+## The `sample` functions
+
+```julia
+sample(
+    rng::Union{AbstractRNG,AbstractVector{<:AbstractRNG}},
+    h::Hamiltonian,
+    κ::HMCKernel,
+    θ::AbstractVector{<:AbstractFloat},
+    n_samples::Int;
+    adaptor::AbstractAdaptor=NoAdaptation(),
+    n_adapts::Int=min(div(n_samples, 10), 1_000),
+    drop_warmup=false,
+    verbose::Bool=true,
+    progress::Bool=false,
+)
 ```
 
+Draw `n_samples` samples using the kernel `κ` under the Hamiltonian system `h`
+
+  - The randomness is controlled by `rng`.
+    
+      + If `rng` is not provided, the default random number generator (`Random.default_rng()`) will be used.
+
+  - The initial point is given by `θ`.
+  - The adaptor is set by `adaptor`, for which the default is no adaptation.
+    
+      + It will perform `n_adapts` steps of adaptation, for which the default is `1_000` or 10% of `n_samples`, whichever is lower.
+  - `drop_warmup` specifies whether to drop samples.
+  - `verbose` controls the verbosity.
+  - `progress` controls whether to show the progress meter or not.
+
+Note that the function signature of the `sample` function exported by `AdvancedHMC.jl` differs from the [`sample`](https://turinglang.org/dev/docs/using-turing/guide#modelling-syntax-explained) function used by `Turing.jl`. We refer to the documentation of `Turing.jl` for more details on the latter.
+
+Note that the function signature of the `sample` function exported by `AdvancedHMC.jl` differs from the [`sample`](https://turinglang.org/dev/docs/using-turing/guide#modelling-syntax-explained) function used by `Turing.jl`. We refer to the documentation of `Turing.jl` for more details on the latter.
+
 ## More types
```

Comment:
If all docstrings are in this section then the name should probably be changed. See the above comment though for a proposal to move some of the docstrings above, in which case the name would remain appropriate.